testCSV.py

```diff
@@ -0,0 +1,24 @@
+from firestore.incidents import Incident
+from cachetools import cached
+from firestore.cachemanager import (INCIDENT_CACHE, INCIDENT_STATS_CACHE,
+                                    flush_cache)
+import csv
+
+
+@cached(cache=INCIDENT_CACHE)
+def get_all_incidents():
+    res = []
+    for incident in Incident.collection.fetch():
+        res.append(incident.to_dict())
+    return res
+
+
+def to_csv():
```

Comment:
Hi Ningzhi,

I'm not sure writing to local file system will work on Google AppEngine, which is a server less environment. Plus even so, when more than one person request download at the same time, the file will conflict with each other.

I would recommend:
1. Try to run AppEngine locally and test if this works;
2. [recommended] Research streaming to download large files, without using local file system.

Thanks!

Li